Sources/JoyfillComponents/Components/Table/CollectionView/CollectionViewTable.swift

```diff
@@ -147,6 +141,23 @@ extension CollectionViewTable: UICollectionViewDelegate, UICollectionViewDataSou
             cell.contentView.subviews.forEach { $0.removeFromSuperview() }
             cell.setupSeparator()
             cell.setDropdownInTableColumn()
+            
+        } else if tableColumnType[tableIndexNo][indexPath.row] == FieldTypes.image {
+            if indexPath.section > 0 {
+                setImageValue(cell: cell, indexPath: indexPath)
+            }
+            if cell.imageCountLabel.labelText == "  Dummy Text" {
```

Comment:
Why is Dummy Text here? What does this do?

Reply:
This " Dummy Text" here is used to hide or unhide image count label in tableImageColumn. As if any cell doesn't have any image then its label gives this value and on this bases I put condition here.